fdbcli/fdbcli.actor.cpp

```diff
@@ -59,31 +59,44 @@ extern const char* getHGVersion();
 
```

Comment:
FYI, this file contains a ton of whitespace changes because I ran git clang-format repeatedly until it stopped making changes.

Point being the whitespace changes are compliant with the checked in .clang-format file